### DISCUSSION TITLE
Clear all applied filters

### DISCUSSION THREAD

**Discussion by chetan-d-mahajan** [microsoft-search/pnp-modern-search#2213] [Q&A] [answered]
I am trying to update/customize the PnP modern search library (v4.7) so as to clear all the applied filters when the user changes a search vertical. May I know if this is feasible in the current implementation ?

**Comment by FranckyC** [comment, marked as answer]
Hi @chetan-d-mahajan, not possible unless you update the core code of the verticals Web part to explicitly fire the "clear" event of the targeted Search Filters Web Part. We use internal events to handle filters operations. See https://github.com/microsoft-search/pnp-modern-search/blob/7710b8a5e9651dadb55f36400ed8c9f77c2f0f97/search-parts/src/components/PanelComponent.tsx#L230 for reference.
